packages/playground/src/tests/r3f-stress-test/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import studio from '@theatre/studio'
+import extension from '@theatre/r3f/dist/extension'
+import {Ticker} from '@theatre/dataverse'
+
+studio.extend(extension)
+studio.initialize()
+
+ReactDOM.render(<App />, document.getElementById('root'))
+
+const raf = Ticker.raf
+
+setInterval(() => {
+  const id = Date.now().toString(36) + '-'
+  performance.mark(id)
```

Comment:
performance.now is more computationally expensive in V8, and it's a lot higher resolution than necessary because we end up just rounding the ticks per second.